tour_site_scraper.rb

```diff
@@ -1,92 +1,189 @@
 require 'rubygems'
 require 'selenium-webdriver'
 require 'pry'
+require 'sqlite3'
 
-MAX_RETRY = 100
-WAIT = Selenium::WebDriver::Wait.new(timeout: 20)
+DB = SQLite3::Database.new( "db_tour_scraper.db" )
```

Comment:
`new("db_tour_scraper.db")`
Ref: [spaces-and-braces](https://github.com/rubocop/ruby-style-guide#spaces-and-braces)